### PR TITLE
i18n(es): Update of `routing.mdx`, `images.mdx`, `imports.mdx` & `markdown-content.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update of multiple pages based on these commits:
  - routing.mdx → 05d022672c4f2a3127283fde055896ca4bbfa9b2
  - images.mdx → 81c1a6413da5a9e7b3b546d2f954bc60ddfb8083
  - imports.mdx → d6e4cbe79b9f495a9611c29dfaa37423cd3fdfbb
  - markdown-content.mdx → 1d43ca26e826d30a174633ba6bab97e060a773c3 & b42a52b6d247c1b7342ce9f7e529338e6811fc58
  - testing.mdx → 903418d7af3458dc6c7ce0dfb503ae4fd86a5921

### CODE REVIEW

File: src/content/docs/es/core-concepts/routing.mdx [hunk 1]
```diff
@@ -373,9 +373,9 @@ const params = Astro.params;
 
 ## Excluyendo páginas
 
-Puedes excluir páginas, o incluso directorios completos de ser generados añadiendo el prefijo (`_`).
+Puedes excluir páginas o directorios de ser generados añadiendo el prefijo (`_`). Los archivos con el prefijo `_` no serán reconocidos por el router y no serán generados en el directorio `dist/`.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/core-concepts/routing.mdx:376]:
```suggestion
Puedes excluir la generación de páginas o directorios añadiendo el prefijo (`_`). Los archivos con el prefijo `_` no serán reconocidos por el router y no serán generados en el directorio `dist/`.
```

File: src/content/docs/es/guides/images.mdx [hunk 2]
```diff
@@ -148,6 +140,14 @@ También debes proporcionar [`width`](/es/guides/integrations-guide/image/#width
 
 [Todas las demás propiedades](/es/guides/integrations-guide/image/#image-) son opcionales.
 
+#### Imágenes locales en `src/`
+
+(atributos requeridos: [`src`](/es/guides/integrations-guide/image/#src), y [`alt`](/es/guides/integrations-guide/image/#alt))
+
+Importa tu imagen en frontmatter y pásala directamente al atributo `src` del componente `<Image />`.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/images.mdx:147]:
```suggestion
Importa tu imagen en el frontmatter y pásala directamente al atributo `src` al componente `<Image />`.
```

File: src/content/docs/es/guides/images.mdx [hunk 3]
```diff
@@ -231,6 +221,16 @@ Debes proporcionar al componente una guía para anchos de imagen y tamaños de p
 
 Aunque [`formats`](/es/guides/integrations-guide/image/#formats) no es necesario, el formato de imagen original de las imágenes remotas es desconocido y no se incluirá por defecto. Si no se proporciona, solo se incluirán `webp` y `avif`.
 
+#### Imágenes locales
+
+(atributos requeridos: [`src`](/es/guides/integrations-guide/image/#src-1), [`widths`](/es/guides/integrations-guide/image/#widths), [`size`](/es/guides/integrations-guide/image/#sizes), [`alt`](/es/guides/integrations-guide/image/#alt-1)
+
+Importa tu imagen en frontmatter y pásala directamente al atributo `<Picture />` del componente `src` .
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/images.mdx:228]:
```suggestion
Importa tu imagen en el frontmatter y pásala directamente al componente `<Picture />` con el atributo `src` .
```

File: src/content/docs/es/guides/images.mdx [hunk 4]
```diff
@@ -231,6 +221,16 @@ Debes proporcionar al componente una guía para anchos de imagen y tamaños de p
...
+
+Debes proporcionar al componente una guía para anchos de imagen y tamaños de pantalla, pero [todas las demás propiedades](/es/guides/integrations-guide/image/#picture-) son opcionales.
+
+Por defecto, el componente `<Picture />` de [`formats`](/es/guides/integrations-guide/image/#formats) incluirá `avif` y `webp` además del formato original de la imagen si no es especificado.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/images.mdx:232]:
```suggestion
Por defecto, el componente `<Picture />` incluirá los [`formatos`](/es/guides/integrations-guide/image/#formats) `avif` y `webp` además del formato original de la imagen si no es especificado.
```

File: src/content/docs/es/guides/markdown-content.mdx [hunk 5]
```diff
@@ -491,6 +492,26 @@ export default defineConfig({
 });
 ```
 
+#### Personalizando un plugin
+
+Para personalizar un plugin, proporciona un objeto de opciones después de él en una matriz anidada.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/markdown-content.mdx:497]:
```suggestion
Para personalizar un plugin, proporciona un objeto de opciones después de él en un array anidado.
```

File: src/content/docs/es/guides/markdown-content.mdx [hunk 6]
```diff
@@ -491,6 +492,26 @@ export default defineConfig({
...
+
+En el ejemplo de abajo se agrega la [opción de encabezado al plugin `remarkToc`](https://github.com/remarkjs/remark-toc#optionsheading) para cambiar dónde es colocada la tabla de contenidos y la [opción `behavior` al plugin `rehype-autolink-headings`](https://github.com/rehypejs/rehype-autolink-headings#optionsbehavior) para agregar la etiqueta de ancla después del texto del encabezado.
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/markdown-content.mdx:499]:
```suggestion
En el ejemplo de abajo se agrega la [opción de encabezado al plugin `remarkToc`](https://github.com/remarkjs/remark-toc#optionsheading) para cambiar dónde es colocada la tabla de contenidos y la [opción `behavior` al plugin `rehype-autolink-headings`](https://github.com/rehypejs/rehype-autolink-headings#optionsbehavior) para agregar una etiqueta ancla después del texto del encabezado.
```

File: src/content/docs/es/guides/markdown-content.mdx [hunk 7]
```diff
@@ -491,6 +492,26 @@ export default defineConfig({
...
+
+```js title="astro.config.mjs"
+import remarkToc from 'remark-toc';
+import { rehypeHeadingIds } from '@astrojs/markdown-remark';
+import rehypeSlug from 'rehype-slug';
+import rehypeAutolinkHeadings from 'rehype-autolink-headings';
+export default {
+  markdown: {
+    remarkPlugins: [],
```
Comment (kevinzunigacuellar) [inline src/content/docs/es/guides/markdown-content.mdx:508]: creo que esto no va?
```suggestion
    remarkPlugins: [],
```

**Review by kevinzunigacuellar — COMMENTED** [review]
gracias @dreyfus92 💜

**Comment by dreyfus92** [comment]
Muchas gracias @kevinzunigacuellar y @agustinmulet por los reviews 🚀

**Comment by Yan-Thomas** [comment]
Thanks @dreyfus92! The Routing and Imports pages had a few changes recently, do you mind adding these to the PR? https://github.com/withastro/docs/pull/2550/files

**Comment by dreyfus92** [comment]
@Yan-Thomas I've added the requested changes, thanks for the heads up. 😄

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @dreyfus92, @agustinmulet & @kevinzunigacuellar for the great work 💜

**Pull request merged**